federatedscope/core/trainers/trainer.py

```diff
@@ -370,15 +370,15 @@ def update(self, model_parameters):
         self.ctx.model.load_state_dict(self._param_filter(model_parameters),
                                        strict=False)
 
-    def evaluate(self, target_data_split_name="test"):
+    def evaluate(self, mode, target_data_split_name="test"):
         with torch.no_grad():
-            super().evaluate(target_data_split_name)
+            super().evaluate(mode, target_data_split_name)
 
         return self.ctx.eval_metrics
 
-    def validate(self, target_data_split_name="val"):
+    def validate(self, mode, target_data_split_name="val"):
```

Comment:
Please unify `evaluate` and `validate`, which have been done in `Trainer`.

Reply:
The `validate` function is deleted.